spring-cloud-gcp-starters/spring-cloud-gcp-starter-sql/src/main/java/org/springframework/cloud/gcp/sql/autoconfig/GcpCloudSqlAutoConfiguration.java

```diff
@@ -131,34 +132,34 @@ public CloudSqlJdbcInfoProvider cloudSqlJdbcInfoProvider(SQLAdmin sqlAdmin,
 	}
 
 	@Bean
+	@ConfigurationProperties(prefix = "spring.datasource")
+	@Primary
```

Comment:
Isn't this Primary going to mess with App Engine? If not, is it really necessary here? I thought with the `@ConditionalOnMissingBeans` there was always only one @Bean.